tests/unit/zmq/test_dual_bind.py

```
@@ -592,13 +617,14 @@ def test_capture_address_disabled(self, tmp_path: Path) -> None:
         cfg = ZMQIPCProxyConfig(path=tmp_path, name="test")
         assert cfg.capture_address is None
 
+    @_skip_on_windows_ipc
     def test_capture_address_enabled(self, tmp_path: Path) -> None:
         cfg = ZMQIPCProxyConfig(path=tmp_path, name="test", enable_capture=True)
         assert "test_capture.ipc" in cfg.capture_address
 
     def test_addr_raises_when_path_is_none(self) -> None:
         cfg = ZMQIPCProxyConfig(name="test")
-        with pytest.raises(ValueError, match="Path is required"):
+        with pytest.raises(ValueError, match="[Pp]ath is required"):
```

Ruff (0.15.14)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)